tests/ctl/core/test_api.py

```diff
@@ -2938,7 +2938,13 @@ def test_api_cannot_delete_default(
             resource_id=resource.fides_key,
             headers=test_config.user.auth_header,
         )
+        print(f"result object: {result}")
+        print(f"result json: {result.json()}")
```

Comment:
Doh, should have removed these. Going to wait for the tests to finish running in CI, then will delete.